tests/test_aaa_authorization_service.rb

```diff
@@ -22,54 +22,72 @@ class TestAaaAuthorizationService < CiscoTestCase
   def setup
     super
     feature_tacacs
+    preconfig_tacacs_server_access(tacacs_groups[0])
```

Comment:
As apposed to the single purpose of setting up the test?  They are quite small already but not that big a deal! :smile: